nav2_bringup/launch/core.launch.py

```diff
@@ -13,8 +13,8 @@ def generate_launch_description():
                 ], period = 1.0),
         launch.actions.TimerAction(
             actions = [
-                #launch_ros.actions.Node( package='nav2_controller_dwb', node_executable='nav2_controller_dwb', output='screen')
-                launch_ros.actions.Node( package='nav2_controller_example', node_executable='dwa_controller', output='screen')
+                launch_ros.actions.Node( package='nav2_controller_dwb', node_executable='nav2_controller_dwb', output='screen')
+                #launch_ros.actions.Node( package='nav2_controller_example', node_executable='dwa_controller', output='screen')
```

Comment:
Can we remove this nav2_controller_example from the launch file?

Reply:
I'll do that in a separate PR to allow this one to go in.

Reply:
Added issue #263